test/unit/deployables/erc20/VotesERC20StakedV1.test.ts

```diff
@@ -36,7 +36,10 @@ async function runExecuteTxAndCheckBalanceDeltasTests(
       if (t.token === 'native') {
         return ethers.provider.getBalance(t.addressToCheck);
       } else {
-        const token = await ethers.getContractAt('IERC20', t.token);
+        const token = await ethers.getContractAt(
+          '@openzeppelin/contracts/token/ERC20/IERC20.sol:IERC20',
```

Comment:
What’s the reason for this change? I don’t see any interface update on the `VotesERC20StakedV1` contract, so I’m wondering why this needs to be updated here in the test file.

Reply:
so without this change, you will get this error:
<img width="935" height="168" alt="image" src="https://github.com/user-attachments/assets/e49a84c1-e491-4cf1-b25c-27925cae67de" />

Since the mock VotingTokenLockupPlans is a flattened version of their file, it declares IERC20. So now we have two artifacts that are IERC20. So we need to specify here which one we are using.